bin/storage_zip_audit.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+# Randomly chooses a Data Den or Glacier object to validate against METS and DB.
+
+use strict;
+use warnings;
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use HTFeed::StorageZipAudit;
+use HTFeed::Log {root_logger => 'INFO, screen'};
+
+die "Specify a feed_backups.storage_name value" unless 1 == scalar @ARGV;
+my $storage_name = $ARGV[0];
+
+my $audit = HTFeed::StorageZipAudit->new($storage_name);
```

Comment:
I think we still need to adjust this to get the storage and ask for the zip auditor rather than constructing StorageZipAudit directly.

Reply:
Except we can't, because the storage needs to be bound to a particular volume, which we don't have right now... I'm not sure I have an elegant solution for getting the right zip audit class right now. Maybe we just specify it in the config per storage back end for the time being.

Reply:
(If only I had remembered to update that file, I might have seen the potential complication! Alas, egg is on my face.)
I actually think the simplest thing, from where I'm sitting, would be to have zip_auditor as a `Storage` class method. Right now in the tests we're shoveling storages all over the place so it's natural to query the instance about an auditor. In terms of starting from a blank slate, I see some fun approaches:

1. Make zip_auditor a Storage class method. Storage subclass need know only the existence of a certain other class.
2. Shove auditor class into config. Replacing spaghetti code with spaghetti config (I am increasingly not a fan of this  seemingly industry-wide push toward increasingly minilanguage-like "configuration goo" [a phrase I have seemingly stolen from Doug McIlroy]).
3. Resurrect my ham-handed "class cluster" initializer from PR part *un*. (Some kind of cluster anyway.)
4. Determine the auditor class dynamically based on the storage class. This has a certain self-contained meta charm but sounds like it could require the StorageZipAudit class to know too much "stuff".

I think I'm going to take a crack at option 1 as I sense Perl fatigue creeping up on me.